setonix/configs/spackuser_pawseystaff/modules.yaml

```diff
@@ -178,9 +178,9 @@ modules:
         "wsclean ~mpi +idg +everybeam": 'astro-applications/{name}/{version}-idg-everybeam'
 
         #performance tests 
-        "hpl ~openmp": 'benchmarking/{name}/{version}'
-        "hpl +openmp": 'benchmarking/{name}/{version}+openmp'
-        osu-micro-benchmarks: 'benchmarking/{name}/{version}'
+        "hpl ~openmp": 'utilities/{name}/{version}'
+        "hpl +openmp": 'utilities/{name}/{version}+openmp'
+        osu-micro-benchmarks: 'utilities/{name}/{version}'
```

Comment:
hpl should stay in benchmarking. I understand the utility of osu but it does say it benchmarking in the name.